server/cloudformation/ecs_test.go

```diff
@@ -59,6 +59,10 @@ func TestECSServiceResource_Create(t *testing.T) {
 		ServiceName:  aws.String("acme-inc-web-dxRU5tYsnzt"),
 		Cluster:      aws.String("cluster"),
 		DesiredCount: aws.Int64(1),
+		DeploymentConfiguration: &ecs.DeploymentConfiguration{
+			MaximumPercent: aws.Int64(100),
+			MinimumHealthyPercent: aws.Int64(80),
+		},
```

Comment:
How come you added this here? We don't currently use DeploymentConfiguration (afaik) in empire. The PR Russell mentioned is working on adding it.

Reply:
The code [here](https://github.com/remind101/empire/blob/master/server/cloudformation/ecs.go#L187) means that unless a `DeploymentConfiguration` with a non-nil `MaximumPercent` and `MinimumHealthyPercent` is present, the ECSServiceResource Update method will panic on accessing the `MaximumPercent.Value()` of a nil `DeploymentConfiguration`

I could have added more checking in the `Update` method, but because I'm not super familiar with either Go or the Empire guts, I opted to try to fix it by only modifying the test, even though that was more extensive.

Reply:
@russellballestrini did you never revert your PR that caused those problems? Sorry Brad, I'd assumed that code had been removed. If that's still in there, master is quite broken ATM.

Reply:
I didn't revert it, I assumed we wanted to fail forward since you had a PR in draft which didn't have the needed changes from the original bad commit. @aengelas what do you prefer we do in this case?